pkg/gds/service.go

```diff
@@ -99,6 +100,11 @@ func New(conf config.Config) (s *Service, err error) {
 		return nil, err
 	}
 
+	// Start the activity publisher
+	if err = activity.Start(conf.Activity); err != nil {
```

Comment:
I guess the bff will do `activity.Subscribe()`?

Reply:
Yeah that makes sense to me.